packages/management-console/src/components/Molecules/DataListItemComponent/DataListItemComponent.tsx

```diff
@@ -526,7 +526,7 @@ const DataListItemComponent: React.FC<IOwnProps> = ({
           <DataListItemCells
             dataListCells={[
               <DataListCell key={1}>
-                <Link to={'/ProcessInstances/' + processInstanceData.id}>
+                <Link to={{pathname:'/ProcessInstances/Process/' + processInstanceData.id, state:{ prev: location.pathname}}}>
```

Comment:
@Sara4994 would it be something like:
```suggestion
                <Link to={{pathname:'/Process/' + processInstanceData.id, state:{ prev: location.pathname}}}>
```

Reply:
that would also mean that we need to add a line here https://github.com/kiegroup/kogito-apps/blob/master/management-console/src/main/java/org/kie/kogito/mgmt/VertxRouter.java#L43
to handle the new path in the UI. 
As well as updating this test https://github.com/kiegroup/kogito-apps/blob/master/management-console/src/test/java/org/kie/kogito/mgmt/VertxRouterTest.java#L17

Reply:
@cristianonicolai done